src/index.js

```diff
@@ -5,10 +5,8 @@ import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <React.StrictMode>
```

Comment:
the React.StrictMode will render our components twice in development mode (not in production) - it's good in development cause we are getting errors and linting the components